proxy/proxy.go

```diff
@@ -22,6 +21,11 @@ type Proxy struct {
 	validatorOIDs []asn1.ObjectIdentifier
 }
 
+const (
+	AttestationTypeHeader string = "X-Flashbots-Ext-OID"
+	MeasurementHeader     string = "X-Flashbots-Measurement"
```

Comment:
should we add TDX in there? like
```suggestion
	MeasurementHeader     string = "X-Flashbots-TDX-Measurement"
```

Reply:
not feeling strongly about it either way